File: lab3/taskA.cpp
#include <iostream>
#include <stack>
#include <array>
using namespace std;

int main()
{
   int length; 
   int arr[] = {2,4,6,7,69}; 
   //create stack
    stack<int> stack;

   printf("Elements in non-reversed order: ");
   for(auto i: arr) // print out the array
   {
        cout<<i<<" ";
        length++;
   }

   for (int i = 0; i < length; i++) // push the array into the stack
   {
        stack.push(arr[i]);
   }

    printf("\nElements in reversed order: ");
    for (int j = length-1; j>= 0; j--) // pop the stack and print out the elements
    {
       arr[j] = stack.top();
       stack.pop();
       cout << arr[j] << " ";
    }

    return 0;
}

File: lab3/taskA.cs
using System;
using System.Collections;

public class TaskA 
{
    public static void Main()
    {
        int arr = new int {3, 6, 9, 12, 69};
        Console.WriteLine(arr);
        // initialize a new stack
        Stack stack = new Stack();

        Console.write("Reverse order: ");
        for (int i = 0; i < arr.length; i++)
        {
            stack.Push(arr[i]);
        }
        for (int j = 0; j < arr.length; j++)
        {
            Console.WriteLine(stack.Pop());
        }
        
    }
}